Consistently bind ActorContext to 'context' in the docs

dwijnand opened the pull request:
... over 'ctx'.

**Original**

Consistently bind ActorContext to 'context' in the docs

... over the lesser used 'context':

    $ rg 'Behavior[s]?.setup' akka-docs/src/test/scala/docs/actor/TypedActorDocSpec.scala akka-actor-typed-tests/ akka-actor-testkit-typed/ | rg ctx | wc -l
          59
    $ rg 'Behavior[s]?.setup' akka-docs/src/test/scala/docs/actor/TypedActorDocSpec.scala akka-actor-typed-tests/ akka-actor-testkit-typed/ | rg context | wc -l
           8
    $ rg 'Behavior[s]?.receive' akka-docs/src/test/scala/docs/actor/TypedActorDocSpec.scala akka-actor-typed-tests/ akka-actor-testkit-typed/ | rg ctx | wc -l
          66
    $ rg 'Behavior[s]?.receive' akka-docs/src/test/scala/docs/actor/TypedActorDocSpec.scala akka-actor-typed-tests/ akka-actor-testkit-typed/ | rg context | wc -l
           5

dwijnand committed:
Consistently bind ActorContext to 'ctx' in the docs

... over the lesser uses 'context':

    $ rg 'Behavior[s]?.setup' akka-docs/src/test/scala/docs/actor/TypedActorDocSpec.scala akka-actor-typed-tests/ akka-actor-testkit-typed/ | rg ctx | wc -l
          59
    $ rg 'Behavior[s]?.setup' akka-docs/src/test/scala/docs/actor/TypedActorDocSpec.scala akka-actor-typed-tests/ akka-actor-testkit-typed/ | rg context | wc -l
           8
    $ rg 'Behavior[s]?.receive' akka-docs/src/test/scala/docs/actor/TypedActorDocSpec.scala akka-actor-typed-tests/ akka-actor-testkit-typed/ | rg ctx | wc -l
          66
    $ rg 'Behavior[s]?.receive' akka-docs/src/test/scala/docs/actor/TypedActorDocSpec.scala akka-actor-typed-tests/ akka-actor-testkit-typed/ | rg context | wc -l
           5

dwijnand committed:
Fix "variable ctx is already defined" warnings

dwijnand commented:
By the way: I'm happy to go the other way if it's preferred.

patriknw reviewed (commented):
Perhaps we should actually use the full name `context`? Not much is gained by using shorter but more obscure `ctx`?

ktoso commented:
`context` is much nicer to write and read... :)

dwijnand committed:
Replace ctx with context, in docs & doc snippets

    fastmod '\bctx\b' 'context' \
        akka-docs/src/main/paradox/typed
        akka-docs/src/test/scala/docs/actor/TypedActorDocSpec.scala \
        akka-actor-typed-tests/ \
        akka-actor-testkit-typed/ \

dwijnand commented:
> Not much is gained by using shorter but more obscure `ctx`?

> `context` is much nicer to write and read... :)

Subjective, but not my fight. 😉

Done.

patriknw commented on akka-actor-typed-tests/src/test/scala/docs/akka/typed/SpawnProtocolDocSpec.scala:
```diff
@@ -63,8 +63,8 @@ class SpawnProtocolDocSpec extends ScalaTestWithActorTestKit with WordSpecLike {
 
-      val greetedBehavior = Behaviors.receive[HelloWorld.Greeted] { (ctx, msg) ⇒
-        ctx.log.info("Greeting for {} from {}", msg.whom, msg.from)
+      val greetedBehavior = Behaviors.receive[HelloWorld.Greeted] { (context, msg) ⇒
```
perhaps change `msg` to `message` while you are at it?

patriknw reviewed (approved):
LGTM, thanks

chbatey reviewed (approved):
LGTM

dwijnand committed:
Replace msg with message, in docs & doc snippets

    fastmod '\bmsg\b' 'message' \
        akka-docs/src/test/scala/docs/actor/TypedActorDocSpec.scala \
        akka-actor-typed-tests/ \
        akka-actor-testkit-typed/
        akka-docs/src/main/paradox/typed/

dwijnand committed:
Fix code alignment

chbatey commented:
Needs a rebase @dwijnand

dwijnand commented:
Self-conflicts, as per my usual. :wink:  Rebased

chbatey commented:
Thanks @dwijnand
